app/models/concerns/foreman_openscap/policy_extensions.rb

```diff
@@ -126,6 +126,11 @@ def assign_hosts(hosts)
       assign_assets hosts.map &:get_asset
     end
 
+    def unassign_hosts(hosts)
+      hosts = hosts.map &:get_asset
+      self.asset_ids = (self.asset_ids - hosts.collect(&:id)).uniq
```

Comment:
minor optimization
```ruby
host_asset_ids = hosts.map { |h| h.get_asset.id }
self.asset_ids = self.asset_ids - host_asset_ids
```

we can obtain ids in one cycle plus I think that uniq is not necessary since we just substract
also note that this will generate SQL query for each host, it may be better to either create get_assets(hosts) method somewhere or use Scaptimony::Asset directly here